demos/portal-embed-demo/README.md

```diff
@@ -0,0 +1,148 @@
+# portal-embed-demo
+This demo demonstrates how Portals can enable a seamless user experience between a website and a third party embed content. Be creative and have fun!!
+<br/>
+<br/>
+## TL;DR;
+- This demo consists of two websites (different origin): 
+  - **PORTALOG**, a blog service, 
+  - and **TTT Archive**, a podcast service.
+- TTT Archive is embedded in one of the PORTALOG's article using **Portals**.
+- The demo shows how portals enables **seamless experience** between to the two websites.
+
+**[This short explainer video (a YouTube Video less than 2 min)](https://youtu.be/4JkipxFVE9k)** is a great place to start from.
+![hero img](https://cdn.glitch.com/98449704-33d8-49b2-88f2-aa6d2aeba5d3%2Fhero_img.png?1556394393372)
+<br/>
+<br/>
+## Running the demo
+### 1. Run the app
+```bash
+$ git clone https://github.com/WICG/portals.git
+$ cd portals/demos/portal-embed-demo
+$ npm install
+$ npm run demo
+```
+Two local servers will start running.
+
+```bash
+$ npm run demo
+📝 PORTALOG has launched: http://localhost:3000?portalport=3001
+🎧 TTT Archive has launched: http://localhost:3001
+```
+
+### 2. Open a browser that supports Portals
+As of May 2019, Chrome Canary is the only platform that supports Portals. You can try out Portals in Chrome Canary by flipping an experimental flag (chrome://flags/#enable-portals).
+
+### 3. Access [http://localhost:3000/](http://localhost:3000/)
```

Comment:
I wanted to make it easy to change the port of the two local servers. 
- `server.js ` has a default port settings but developers might want to change the port being used: https://github.com/WICG/portals/pull/105/commits/af7a1e8b2d8bd7f64b42ebc1515cd6cfb06f8b68#diff-ffc49d078375ee2224fd41a7bbb1d456R3
- and `public/js/portalog/portals-controller.js` checks if it's using other than the default 3001 for TTT Archive and replaces the URL to use for portals: https://github.com/WICG/portals/pull/105/commits/af7a1e8b2d8bd7f64b42ebc1515cd6cfb06f8b68#diff-aff8d943c491a10805f3c6c4b735b032R201 

Since in this application, both `server.js` and portalog needs to know the port used for TTT Archive, the only thing I could think of to easily avoid hard coding was to pass the port with a query parameter.

Does this make sense?

Reply:
Sorry for not being clear. I'm wondering if the instructions should point to http://localhost:3000/?portalport=3001, to be consistent with what the npm command spits out, or  alternatively, instead instruct readers to click the link npm emits.

Reply:
Got you point. Changed it to access ` http://localhost:3000/?portalport=3001`.